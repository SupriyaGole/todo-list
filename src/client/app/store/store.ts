import { configureStore } from "@reduxjs/toolkit";

import todoList from "../../components/reducer";

export const store = configureStore({
  reducer: {
    todoList: todoList,
  },
});
