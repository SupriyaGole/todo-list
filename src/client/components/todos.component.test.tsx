import React from "react";

import { render } from "@testing-library/react";
import { Provider } from "react-redux";

import { store } from "../app/store/store";
import Todos from "./todos.component";

describe("Todos", () => {
  it("should render a component", () => {
    const { asFragment } = render(
      <Provider store={store}>
        <Todos />
      </Provider>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
