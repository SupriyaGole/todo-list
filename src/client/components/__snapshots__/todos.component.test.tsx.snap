// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todos should render a component 1`] = `
<DocumentFragment>
  <div
    class="todos"
  >
    <header>
      <h1>
        TODO - Daily Task Manager
      </h1>
    </header>
    <div
      class="main-page"
    >
      <div
        class="todo-creator"
      >
        <input
          id="todo-input"
          placeholder="add your first todo task"
          type="text"
          value=""
        />
        <button
          data-testid="addTodo"
          disabled=""
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
